quazip/quazipfile.cpp

```diff
@@ -340,10 +340,10 @@ bool QuaZipFile::open(OpenMode mode, const QuaZipNewInfo& info,
     else
         zipClearFlags(p->zip->getZipFile(), ZIP_WRITE_DATA_DESCRIPTOR);
     p->setZipError(zipOpenNewFileInZip3_64(p->zip->getZipFile(),
-          p->zip->getFileNameCodec()->fromUnicode(info.name).constData(), &info_z,
+          qUtf8Printable(info.name), &info_z,
```

Comment:
This won't do. `qUtf8Printable` was introduced in Qt 5.4. QuaZIP supports backward compatibility with Qt 4.x. Better replace with `toUtf8().constData()` here and in other places too.

Reply:
Ok it's easy.